### ISSUE TITLE
Gh command clashes with GitHub cli short command

### ISSUE THREAD

**Issue by indcoder** [MichaelJolley/ps-alias#53]
If you install GitHub CLI, then you can use it since it also uses the same command [gh]

**Comment by michaeljolley** [comment]
That's true, but I don't think `gh` has the same action, does it? In our case, `gh` will open the repo in a browser.

This module may conflict with the GitHub CLI for that reason.
